src/SayMoreTests/ApplicationContainerTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Linq;
+using NUnit.Framework;
+using SayMore;
+using SayMore.Model.Files;
+using SayMore.Properties;
+
+namespace SayMoreTests
+{
+	[TestFixture]
+	public class ApplicationContainerTests
+	{
+		[Test]
+		public void ComponentRoles_ContainsOralTranslationRole_HasCorrectRenamingSuffix()
```

Comment:
Normal three-part naming convention is <method or property under test>_<condition>_<expected result>